impl.go

```diff
@@ -84,17 +84,23 @@ func findInterface(iface string, srcDir string) (path string, id string, err err
 		panic(err)
 	}
 	if len(f.Imports) == 0 {
-		return "", "", fmt.Errorf("unrecognized interface: %s", iface)
-	}
-	raw := f.Imports[0].Path.Value   // "io"
-	path, err = strconv.Unquote(raw) // io
-	if err != nil {
-		panic(err)
+		if !strings.Contains(iface, ".") {
+			path = "."
+			id = iface
+		} else {
+			return "", "", fmt.Errorf("unrecognized interface: %s", iface)
```

Comment:
If you reorder this and unindent some it'll be easier to read. Comment, too.

```go
// If we were given an identifier with no "." in it, and goimports couldn't resolve it,
// assume it is from the current package.
if strings.Contains(iface, ".") {
  return "", "", fmt.Errorf("........")
}
return ".", iface, nil
```

Then the else below can be removed and the contents unindented as well.